content/join.hbs

```diff
@@ -91,7 +91,11 @@
 			<p class="has-text-centered">To apply to AFROTC and complete the application checklist, follow this guide.</p>
 			<p class="has-text-centered"><a class="button is-large future-interest" href="files/WINGS Application Guide.pdf">WINGS Application Guide</a></p>
 
-			<h1>Join in <span class="semester-text">Spring 2024</span> or later</h1>
+<<<<<<< HEAD

```

Comment:
Another merge conflict that got left over. I think you want to keep the `<h1>Join in <span class="semester-text">Spring 2025</span> or later</h1>` and delete the rest